unime/src/routes/+layout.svelte

```diff
@@ -57,46 +59,121 @@
       goto(`/prompt/${type}`);
     }
   }
+
+  interface DevModeButton {
+    stringIcon?: string;
+    svelteIcon?: typeof SvelteComponent<SvelteHTMLElements['svg']>;
+    onClick: () => void;
+  }
+
+  function createDevButtons(): DevModeButton[] {
+    const backBtn: DevModeButton = {
+      svelteIcon: ArrowLeft,
+      onClick: () => history.back(),
+    };
+
+    const resetBtn: DevModeButton = {
+      svelteIcon: Trash,
+      onClick: () => dispatch({ type: '[App] Reset' }),
+    };
+
+    const ferrisBtn: DevModeButton = {
```

Comment:
Please be consistent in naming and either spell it out: `ferrisButton` or remove it completely: `ferris` (since it is defined in a `const devButtons` and it's clear from the context). There is no need for random abbreviations such as `Btn`.

Reply:
Its inside a function that literally tells it what it is.... There is no reason to the change spelling here.

Reply:
Yes, you are right. In this particular case it might not matter. I just believe that "naming things" is a very important thing when building software, so I tried to stay consistent.

Reply:
Sure I will do that in the future